content/docs/1.1.0/concepts.md

```diff
@@ -90,9 +90,10 @@ The Longhorn CSI driver takes the block device, formats it, and mounts it on the
 The Kubernetes CSI Driver images are:
 
 - CSI Attacher: quay.io/k8scsi/csi-attacher:v2.0.0
-- CSI Provisioner: quay.io/k8scsi/csi-provisioner:v1.4.0
+- CSI Provisioner: quay.io/k8scsi/csi-provisioner:v1.6.0
```

Comment:
We can remove the mention of driver images since all of them are covered by https://github.com/longhorn/longhorn/blob/master/deploy/longhorn-images.txt

Reply:
removed the specific images and instead linked to the airgap section.